app/(root)/page.tsx

```
@@ -1,7 +1,30 @@
+
   return (
     <>
-      <div className="px-2"></div>
+      <div className="px-2">
+        <h1>
+          {data &&
+            data?.results.map((item) => (
+              <Link href={`/questions/${item.id}`} key={item.id}>
+                {item?.properties?.Name.title[0].plain_text}
+              </Link>
+            ))}
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)